src/App.tsx

```diff
@@ -1,203 +1,97 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+import cn from 'classnames';
+
+import { Todo } from './types/Todo';
+import { ErrorType } from './types/ErrorType';
+
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { FilterType } from './types/FilterType';
+import { getFilteredTodos } from './utils/getFilteredTodo';
+import { Footer } from './components/Footer';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filter, setFilter] = useState(FilterType.all);
+  const [errorMessage, setErrorMessage] = useState<ErrorType | null>(null);
```

Comment:
u can make `default` value in ur enum `ErrorType.Default === ''`
```suggestion
  const [errorMessage, setErrorMessage] = useState<ErrorType>(ErrorType.Default);
```